### ISSUE TITLE
Connector dependency reporter does not include normalization changes

### ISSUE THREAD

**Issue by tuliren** [airbytehq/airbyte#17897]
## Summary
- Currently the `report-connectors-dependency` action does not include normalization changes.
- It should when there is any change under `base-normalization`.

**Comment by erohmensing** [comment]
@tuliren can you help me understand this one better? I thought it was saying that `base-normalization` changes weren't registered by the script looking for dependencies. However I made a change similar to the one Greg made in `airbyte-integrations/bases/base-normalization/normalization/transform_config/transform.py` to test, and ended up with the same results as Greg:

![image](https://user-images.githubusercontent.com/22333512/199276046-54b4a5b0-0338-4132-ac16-111769b19004.png)

Do you mean that the action should report changes to normalization itself, as well as the sources and destinations affected?

**Comment by tuliren** [comment]
> Do you mean that the action should report changes to normalization itself, as well as the sources and destinations affected?

Yes. The version of the normalization also needs to be bumped. The PR I linked in the description did not bump the version for the normalization module, and caused a problem (if I remembered correctly).

**Comment by erohmensing** [comment]
So whenever there are changes in the normalization module, we need to check if the version for the normalization module was bumped, and if not, tell the user they need to bump it (or automate the bumping)?

That would make sense to me as something to implement, not sure that it makes sense for the connector dependency reporter to be performing that though

**Comment by evantahler** [comment]
@edgao can you please help us choose what to do?

**Comment by evantahler** [comment]
Grooming - The list should also include the normalization containers (e.g. postgres-normalization)

**Comment by erohmensing** [comment]
Looks like the original issue was for avoiding issues like this (the "published in") -- essentially saying that if you make changes to `base-normalization`, you need to publish those changes in order for them to be live.

<img width="1020" alt="image" src="https://user-images.githubusercontent.com/22333512/201147398-cdfd89d6-159d-42ae-8259-33b519c0aade.png">
